starters/qwik-graphql-tailwind/src/routes/counter-example/index.tsx

```diff
@@ -10,6 +11,11 @@ export default component$(() => {
       <div className="py-5">
         <Counter />
       </div>
+      <div className="my-2.5">
```

Comment:
should we use `class` instead of `className`?

Reply:
hmm, I think that will be a general update on the app. this ticket is more focus on the link

Reply:
this has been resolved in a diff pr